spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -77,6 +77,13 @@ When("I click the back link", () => {
   cy.clickBackLink();
 });
 
+When("I add a csv file to the file input", () => {
+  cy.get("#partnership-csv-upload-csv-field").attachFile({
```

Comment:
This doesn't look very generic. We either need to make this step reusable, or make it sound like it isn't

Reply:
I was just happy to get it working...will have a look. It would need to have an id field and a fixture reference to be reusable

Reply:
It's for it not to be reusable, but maybe call it "I upload the school urn csv" or something specific?